coalib/settings/SectionFilling.py

```diff
@@ -28,6 +29,7 @@ def fill_settings(sections, acquire_settings, log_printer):
     global_bears = {}
 
     for section_name, section in sections.items():
+        section.aspects = extract_aspects_from_section(section)
```

Comment:
I am not sure whether to declare `section.aspects` in the class Section `__init__` declaration because all section object is guaranteed to run this `fill_settings` function anyway.